util/util.go

```diff
@@ -685,31 +686,50 @@ func IsKubernetesDefaultToleration(toleration v1.Toleration) bool {
 	return false
 }
 
-func AreIdenticalTolerations(oldTolerations, newTolerations map[string]v1.Toleration) bool {
-	// modified existing tolerations
-	for name := range oldTolerations {
-		if !IsKubernetesDefaultToleration(oldTolerations[name]) && !reflect.DeepEqual(newTolerations[name], oldTolerations[name]) {
-			return false
-		}
+func GetAnnotation(obj runtime.Object, annotationKey string) (string, error) {
```

Comment:
I think it's better to put these 2 functions in `datastore/kubernetes`

Reply:
I think that `datastore/kubernetes` contains functions that make requests to Kube API. However, `GetAnnotation()` and `SetAnnotation()` function here only extract the data from its input parameters. Moreover, the 2 functions are used in the upgrade part which doesn't have access to DS. So, I think that they should be in `util`
How do you think?

Reply:
OK. It makes sense.